README.md

```diff
@@ -1 +1,150 @@
-# Leaderboard-project
\ No newline at end of file
+# Leaderboard
+
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+</h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Visit my website](https://kevin-mena.github.io/Personal-Portfolio-Website/)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+\*\*[Leaderboard] is a simple app where users can add names and scores of games and get stored in a Leaderboard API/
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Markup</summary>
+  <ul>
+    <li><a href="https://https://developer.mozilla.org">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Styles</summary>
+  <ul>
+    <li><a href="https://https://developer.mozilla.org">CSS</a></li>
+  </ul>
+</details>
+
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Local Storage]**
+- **[APIs]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo" ></a>
+
+- [Live Demo Link](Coming soon...)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+A text-editor like VS code or Sublime Editor and a github account.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project with git bash or github desktop.
+
+### Usage
+
+To run the project, execute the following command:
+$git clone to clone the project to your desktop.
+
+### Run tests
+
+To run tests, run the following command:
+bin/rails test test/models/article_test.rb
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [Optional] Kindly consider making your getting started section more descriptive. Add steps to guide a visitor to easily use your project. This will make your work more professional.
- 
![image](https://user-images.githubusercontent.com/94551484/223807194-4cac0d3a-b2d8-41c4-b6a4-ba25ef099fd8.png)